eng/data-plane.proj

```diff
@@ -2,8 +2,11 @@
   <ItemGroup>
     <!-- Issue https://github.com/Azure/azure-sdk-for-net/issues/5089 some KeyVault tests don't work on linux -->
     <ExcludeDataPlaneProjects Include="$(RepoSrcPath)\KeyVault\data-plane\**\Microsoft.Azure.KeyVault.Tests.csproj" Condition="'$(OS)' != 'Windows_NT'" />
```

Comment:
Since this project has been removed you can delete this line now.

Reply:
Ok.